ambari-server/src/main/resources/custom_actions/scripts/mpack_packages.py

```diff
@@ -86,7 +86,7 @@ def actionexecute(self, env):
     # Build structured output with initial values
     self.structured_output = {
       'package_installation_result': 'FAIL',
-      'mpack_id': command_repository.mpack_id
+      'mpackId': command_repository.mpack_id
```

Comment:
Hah! Too funny on this one. So, did the listener dump log statements saying it didn't know what to update? If not, maybe it should to help track down if this data for some reason isn't in the response.

Reply:
Correct: because mpackId was null.  That was my clue that something was off.  But that was only after I updated the short-cut that didn't invoke the listener.  Multiple "issues" for the response not being read.